### ISSUE TITLE
Problems in research results

### ISSUE THREAD

**Issue by ShuaikangXie** [fukexue/RGM#19]
Hello, the author, seeing your new research results on RGM in 2022, the RR reached 95.5%. I am very interested in your research in this field. Will your code be open source?

**Comment by fukexue** [comment]
Yes, I will open source the code. But I am currently busy with graduation and haven't finished organizing yet.
